pkg/manager-nnf/allocation_policy.go

```diff
@@ -166,8 +166,16 @@ func (p *SpareAllocationPolicy) CheckCapacity() error {
 			return fmt.Errorf("Insufficient drive count. Available %d", len(p.storage))
```

Comment:
Seems like 16 should be a constant and the instances below should use that constant.